lib/locales/ta.ts

```diff
@@ -0,0 +1,998 @@
+import { Locale } from '../types/locale';
+
+export const locale: Locale = {
+  id: 'ta',
+
+  seasons: {
+    advent: {
+      season: 'திருவருகைக்காலம்',
+      weekday: '$t(weekdays:{{dow}}, capitalize) - $t(ordinals:{{week}}) திருவருகைக்கால வாரநாள்',
+      sunday: '$t(ordinals:{{week}}, capitalize) திருவருகைக்கால ஞாயிறு',
+      privileged_weekday: '$t(months:11, capitalize) {{day}}',
+    },
+
+    christmas_time: {
+      season: 'கிறிஸ்து பிறப்பு காலம்',
+      day: '$t(weekdays:{{dow}}, capitalize) கிறிஸ்து பிறப்பு காலம்',
+      octave: '{{count}}ஆம் நாள் கிறிஸ்து பிறப்பின் எண்கிழமை',
+      second_sunday_after_christmas: 'கிறிஸ்து பிறப்பிற்குப் பின் இரண்டாம் ஞாயிறு',
+      before_epiphany: 'திருக்காட்சிக்கு முன் - $t(months:0, capitalize) {{day}}',
+      after_epiphany: 'திருக்காட்சிக்கு பின் $t(weekdays:{{dow}}, capitalize)',
```

Comment:
Don't you want to add a hyphen (`-`), like you did in the value of `before_epiphany`?

```suggestion
      after_epiphany: 'திருக்காட்சிக்கு பின் - $t(weekdays:{{dow}}, capitalize)',
```